modular_skyrat/code/modules/jobs/job_types/mining_foreman.dm

```diff
@@ -0,0 +1,118 @@
+	department_flag = CIVILIAN
+	faction = "Station"
+	total_positions = 2
+	spawn_positions = 2
+	supervisors = "the quartermaster and the mining foreman."
+	selection_color = "#ca8f55"
+	outfit = /datum/outfit/job/miner_engineer
+	access = list(ACCESS_ENGINE, ACCESS_ENGINE_EQUIP, ACCESS_TECH_STORAGE, ACCESS_CONSTRUCTION, ACCESS_MAINT_TUNNELS, ACCESS_MAILSORTING, ACCESS_CARGO, ACCESS_CARGO_BOT, ACCESS_MINING,
+				ACCESS_MINING_STATION, ACCESS_MINERAL_STOREROOM)
+	minimal_access = list(ACCESS_ENGINE, ACCESS_ENGINE_EQUIP, ACCESS_TECH_STORAGE, ACCESS_CONSTRUCTION, ACCESS_MAINT_TUNNELS, ACCESS_MINING, ACCESS_MINING_STATION, ACCESS_MAILSORTING, ACCESS_MINERAL_STOREROOM) //Construction for Aux base
+	paycheck = PAYCHECK_MEDIUM //This role is an engineer focused around the Aux base and surface.
+	paycheck_department = ACCOUNT_CAR
+
+	display_order = JOB_DISPLAY_ORDER_MINER_ENGINEER
+
+	threat = 1
+
+/datum/job/mining_engineer/radio_help_message(mob/M)
+	to_chat(M, "<span class='userdanger'>You are a technician based on maintaining and expanding the surface-side facilities. Especially the Auxiliary base. You are not paid to maintain the station.</span>")
+	to_chat(M, "<span class='userdanger'>In an emergency you are considered to have the same training as station engineers, but the CE and station engineers have a higher authority than you for anything related to the station proper. </span>")
+
+
+/datum/outfit/job/miner_engineer
+	name = "Mining Technician"
+	jobtype = /datum/job/mining_engineer
+
+	belt = /obj/item/storage/belt/utility/full/engi
+	ears = /obj/item/radio/headset/headset_mining_engie
+	uniform = /obj/item/clothing/under/rank/engineering/engineer
+	shoes = /obj/item/clothing/shoes/workboots
+	gloves = /obj/item/clothing/gloves/color/black
+	head = /obj/item/clothing/head/hardhat
+	l_pocket = /obj/item/pda/engineering
+	r_pocket = /obj/item/t_scanner
+	backpack = /obj/item/storage/backpack/industrial
+	satchel = /obj/item/storage/backpack/satchel/eng
+	duffelbag = /obj/item/storage/backpack/duffelbag/engineering
+	box = /obj/item/storage/box/engineer
+	pda_slot = SLOT_L_STORE
+	backpack_contents = list(
+		/obj/item/modular_computer/tablet/preset/advanced=1,\
+		/obj/item/flashlight/seclite=1,\
+		/obj/item/kitchen/knife/combat/survival=1,\
+		/obj/item/stack/marker_beacon/ten=1)
+
+
+/obj/item/encryptionkey/headset_cargo/mining_engineer
+	name = "mining construction radio encryption key"
+	icon_state = "cargo_cypherkey"
+	channels = list(RADIO_CHANNEL_SUPPLY = 1, RADIO_CHANNEL_ENGINEERING = 1)
+
+/obj/effect/landmark/start/mining_foreman
+	name = "Mining Foreman"
+	icon_state = "Shaft Miner"
+
+/obj/effect/landmark/start/mining_engineer
+	name = "Mining Technician"
+	icon_state = "Station Engineer"
+
+/obj/item/radio/headset/headset_mining_engie
+	name = "mining technician radio headset"
+	desc = "A headset used by mining technicians."
+	icon_state = "cargo_headset"
+	keyslot = new /obj/item/encryptionkey/headset_cargo/mining_engineer
```

Comment:
newline, please

Reply:
Should be good now. If not let me know please!